ipywidgets/widgets/widget.py

```diff
@@ -717,9 +722,20 @@ def _ipython_display_(self, **kwargs):
                     'model_id': self._model_id
                 }
             }
-            display(data, raw=True)
+            return data
 
-            self._handle_displayed(**kwargs)
+    def _ipython_display_(self, **kwargs):
```

Comment:
@jasongrout the definition of `_ipython_display_` could be behind a `if` statement.

```python
    if ipython_version_info < (6,  1):
        def _ipython_display_(self, **kwargs):
        """Called when `IPython.display.display` is called on a widget.
        Note: only defined for IPython <6.1.
        """
        data = self._repr_mimebundle_(**kwargs)
        if data:
            display(data, raw=True)
```